lib/debug/commands/temperature_commands.cpp

```diff
@@ -0,0 +1,52 @@
+#include "temperature_commands.hpp"
+
+#include <sensors/temperature.hpp>
+
+namespace hyped::debug {
+
+core::Result TemperatureCommands::addCommands(core::ILogger &logger,
+                                              std::shared_ptr<Repl> repl,
+                                              toml::v3::node_view<toml::v3::node> config)
+{
+  const auto optional_bus = config["bus"].value<std::uint8_t>();
+  if (!optional_bus) {
+    logger.log(core::LogLevel::kFatal, "Invalid I2C bus name");
+    return core::Result::kFailure;
+  }
+  const auto bus          = *optional_bus;
+  const auto optional_i2c = repl->getI2c(bus);
+  if (!optional_i2c) {
+    logger.log(core::LogLevel::kFatal, "Error creating I2C bus");
+    return core::Result::kFailure;
+  };
+  const auto i2c              = std::move(*optional_i2c);
+  const auto optional_address = config["address"].value<std::uint8_t>();
+  if (!optional_address) {
+    logger.log(core::LogLevel::kFatal, "Invalid address");
+    return core::Result::kFailure;
+  }
+  const auto address                     = std::move(*optional_address);
+  const auto optional_temperature_sensor = sensors::Temperature::create(
+    logger, i2c, 0, static_cast<sensors::temperatureAddress>(address));
```

Comment:
Shouldn't this be bus 2, as defined in the debug.toml?

Reply:
that's the mux channel. we'll merge this, then when the accelerometer PR gets merged there will be a cleanup